deck.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'flashcard'
+require_relative 'card_parser'
+
+class Deck
+  attr_accessor :deck, :flashcards
+
+  include Parsable
+
+  def initialize(flashcards)
+    @flashcards = flashcards
+  end
+
+  def clear_deck
+    self.deck.clear
+  end
+end
```

Comment:
This is a nice concise class, but I would expect more methods in here dealing with Deck logic and less in the controller.